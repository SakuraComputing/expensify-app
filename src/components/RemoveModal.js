import React from 'react';
import Modal from 'react-modal';

const RemoveModal =({}) => (

    <Modal
        contentLabel="Remove Expense"
    >

        <h3>Remove this expense?</h3>
        <button>Remove</button>
        <button>Cancel</button>
    </Modal>
);
export default RemoveModal;
