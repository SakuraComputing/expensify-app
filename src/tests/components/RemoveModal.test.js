import React from 'react';
import { shallow } from 'enzyme';
import RemoveModal from '../../components/RemoveModal';


test('should render the remove modal form correctly', () => {
   const wrapper = shallow(<RemoveModal/>);
   expect(wrapper).toMatchSnapshot();
});
