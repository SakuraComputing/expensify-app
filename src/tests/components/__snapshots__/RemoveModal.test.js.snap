// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the remove modal form correctly 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  closeTimeoutMS={0}
  contentLabel="Remove Expense"
  isOpen={false}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnOverlayClick={true}
>
  <h3>
    Remove this expense?
  </h3>
  <button>
    Remove
  </button>
  <button>
    Cancel
  </button>
</Modal>
`;
